recipes/quickjs/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.0.0)
```

Comment:
Usually the patch version number is omitted in the CMake minimum version.

Reply:
Thank you for your comment.
I didn't know it.
Fixed.